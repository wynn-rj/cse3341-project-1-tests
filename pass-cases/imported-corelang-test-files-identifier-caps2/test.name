identifier caps2 (Imported from corelan-test-files)
