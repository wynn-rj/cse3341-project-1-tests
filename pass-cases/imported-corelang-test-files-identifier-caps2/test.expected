program
  int x,X;
begin
  x:=3;
  X:=4;
  output x;
  output X;
end
3
4
